ui_framework/components/_common_styles.scss

```diff
@@ -15,3 +15,10 @@
 body {
   font-family: $globalFontFamily;
 }
+
+/**
+ * 1. Required for IE11.
+ */
+main {
+  display: block; /* 1 */
+}
```

Comment:
Are you sure, this is still needed? Running the task (or also `uiFramework:build` doesn't change any files managed by git. Do I need to copy the compiled files somewhere?